backend/cli_commands/SyncCommand.py

```diff
@@ -87,6 +85,43 @@ def sync_files(mission_path, mission):
                         )
                     except Exception as e:
                         logging.error(f"Error processing {mcap_path}: {e}")
+                else:
+                    file = File.objects.get(file=mcap_path)
```

Comment:
```suggestion
                    file = File.objects.get(file=mcap_path)
                # generate videos for new file
                generate_videos(mcap_path)
```
here you should generate the videos